themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -27,24 +27,25 @@
   }
 ?>
 <button class="close-offcanvas btn btn-link" data-toggle="offcanvas"><?=$this->transEsc('navigate_back') ?></button>
-<?php if ($results->getResultTotal() > 0): ?>
-  <h2><?=$this->transEsc($this->slot('side-facet-caption')->get('Narrow Search')) ?></h2>
-<?php endif; ?>
-<?php $checkboxFilters = $this->recommend->getCheckboxFacetSet(); ?>
-<?php $checkboxesShown = false; ?>
-<?php if (count($checkboxFilters) > 0):
+<?php
+  $checkboxFilters = $this->recommend->getCheckboxFacetSet();
+  $checkboxesShown = false;
```

Comment:
I'd like this to be renamed to something that doesn't imply the checkboxes have already been shown.

Reply:
Good point -- changed to `$hasVisibleCheckboxes`.